nengo_gui/static/components/value.js

```diff
@@ -114,6 +114,26 @@ Nengo.Value = function(parent, sim, args) {
     this.on_resize(this.get_screen_width(), this.get_screen_height());
     this.axes2d.axis_y.tickValues([args.min_value, args.max_value]);
     this.axes2d.fit_ticks(this);
+
+    this.colors = Nengo.make_colors(6);
+    this.color_func = function(d, i) {return self.colors[i % 6]};
+    this.legend = document.createElement('div');
+    this.legend.classList.add('legend');
+    this.div.appendChild(this.legend);
+
+    this.legend_labels = args.legend_labels || [];
+    console.log(this.legend_labels);
+    if (this.legend_labels.length !== this.n_lines) {
+        // fill up the array with temporary labels
+        for (var i=this.legend_labels.length; i<this.n_lines; i++) {
+            this.legend_labels[i] = "label_" + i;
```

Comment:
Could the defaults be "0, 1, 2, ..." rather than "label_0, label_1, label_2, ..."?